asciidoxy/collect.py

```diff
@@ -245,6 +245,12 @@ async def _download_files(self, package_dir: Path, session: aiohttp.ClientSessio
 
         jobs = []
         for file_name in self.file_names:
+            if "{version}" in file_name and "{name}" in file_name:
```

Comment:
First if statement is required, otherwise it will fail when both options are provided.

Reply:
Not sure if it is maybe better to use `elif` here and just put empty else at the end to follow conventions.

Reply:
I think that the nicest combination will be:
```
if version and name:
elif version:
elif name:
```
And no need for an else, because then `file_name` is already correct